optimum/exporters/openvino/__main__.py

```diff
@@ -364,7 +369,7 @@ class StoreAttr(object):
 
             if tokenizer is not None:
                 try:
-                    export_tokenizer(tokenizer, output)
+                    export_tokenizer(tokenizer, output / OV_TOKENIZER_FOLDER)
```

Comment:
let's drop adding this subfolder.

Export to OpenVINO should just replace original files / models with OpenVINO ones, without changing file structure.

Reply:
Moved OV tokenizer to the root folder again.